.github/workflows/datatransport.yml

```diff
@@ -37,6 +37,20 @@ jobs:
     - name: Setup project and Test Catalyst
       run: scripts/third_party/travis/retry.sh scripts/test_catalyst.sh ${{ matrix.pod }} test
 
+  watchos-testapp:
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        pod: [GoogleDataTransport, GoogleDataTransportCCTSupport]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Prereqs
+      run: scripts/install_prereqs.sh ${{ matrix.pod }} watchOS xcodebuild
+    - name: Setup project and test watchOS test app
```

Comment:
nit: Setup project and build watchOS test app

Reply:
Thanks, Paul.